dataikuapi/dss/ml.py

```diff
@@ -792,7 +781,7 @@ def condition(self):
     @condition.setter
     def condition(self, condition):
         if not isinstance(condition, DSSMLAssertionCondition):
-            raise ValueError('Wrong type for assertion condition: {}.format(type(condition)))
+            raise ValueError('Wrong type for assertion condition: {}.format(type(condition))')
```

Comment:
Does not look right :)
```suggestion
            raise ValueError('Wrong type for assertion condition: {}'.format(type(condition)))
```

Reply:
Thanks !